code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -76,7 +76,7 @@
 /obj/item/gun/energy/kinetic_accelerator/cyborg
 	holds_charge = TRUE
 	unique_frequency = TRUE
-	max_mod_capacity = 80
+	max_mod_capacity = 100
```

Comment:
I suggest you write "//skyrat edit" after the mod cap modified.

Reply:
Appreciated! however I wasn't aware this type of change was attempted before, and since the new Crusher just recently got re-added, believe its for the best to leave this value alone for now at least.

Reply:
Yeah, that was sad to me having my PR buffing the borg KA rejected, let's hope seeing the borgs wielding plasma cutters soon, I'd love to wield that as minerborg.